storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -37,6 +37,11 @@ include::modules/persistent-storage-csi-vsphere-stor-policy.adoc[leveloffset=+1]
 :FeatureName: vSphere
 include::modules/persistent-storage-csi-tp-enable.adoc[leveloffset=+1]
 
+[NOTE]
+====
+In the PV specifications, do not use key `storage.kubernetes.io/csiProvisionerIdentity` in `csi.volumeAttributes` because this key indicates dynamically provisioned PVs.
```

Comment:
```suggestion
Do not use the key `storage.kubernetes.io/csiProvisionerIdentity` in the `csi.volumeAttributes` field of your persistent volume (PV) specification. This key creates a dynamically provisioned PV.
```

As a reader, I am confused by the mention of the `csi.volumeAttributes` field, because it is not referenced anywhere else in the assembly. Also, why is it a problem that this field assigns is a dynamic provision? How do I set or unset this field?

Lastly, I think we would normally have the admonition block in the same module as the procedure, but it isn't clear to me if the procedure above is directly related to the note.

Reply:
@jsafrane @duanwei33 could you take a look at the first statement>> As a reader, I am confused by the mention of the csi.volumeAttributes field, because it is not referenced anywhere else in the assembly. Also, why is it a problem that this field assigns a dynamic provision? How do I set or unset this field?